<launch>
  <arg name="image_topic" default="/usb_cam/image_raw"/>  
  <arg name="calibration_path" default="$(find camera_pkg)/param/camera.yaml"/>  
  <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="video_device" value="/dev/video0" />
    <param name="image_width" value="640" />
    <param name="image_height" value="480" />
    <param name="pixel_format" value="yuyv" />
    <param name="camera_frame_id" value="usb_cam" />
    <param name="io_method" value="mmap"/>
  </node>
  <!-- <node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
    <remap from="image" to="/usb_cam/image_raw"/>
    <param name="autosize" value="true" />
  </node> -->
  <node name="aruco_detector" pkg="camera_pkg" type="aruco_detector" respawn="false" output="screen">
    <param name="image_topic" value="$(arg image_topic)" />
    <param name="calibration_path" value="$(arg calibration_path)" />
  </node>
</launch>
